src/installer.rs

```diff
@@ -45,7 +45,7 @@ pub fn install() -> ! {
     process::exit(0);
```

Comment:
what do you think about just `bail!`ing here and returning a Result from anyhow instead of the never return type? would need to update the callers, but this is an error I think we'd like to be able to diagnose from the automated user reporting I'm working on.